set.mm

```diff
@@ -47195,12 +47207,12 @@ This theorem is proved directly from set theory axioms (no set theory
        for ` z ` throughout the proof, the proof would fail.  (Contributed by
        NM, 7-Nov-2006.) $)
     dtru $p |- -. A. x x = y $=
-      ( vw vz weq wn wex wal wel wa el ax-nul sp eximii exdistrv mpbir2an con3d
-      ax9 ax7 spimev com12 con3dimp 2eximi wi equequ2 notbid syl6bi a1d pm2.61i
-      exlimivv mp2b exnal mpbi ) ABEZFZAGZUNAHFACIZADIZFZJZDGCGZCDEZFZDGCGUPVAU
-      QCGUSDGACKUSAHUSDDALUSAMNUQUSCDOPUTVCCDUQVBURVBUQURCDARUAUBUCVCUPCDDBEZVC
-      UPUDVDVCCBEZFZUPVDVBVEDBCUEUFVFUOACACEUNVEACBSQTUGVDFZUPVCVGUOADADEUNVDAD
-      BSQTUHUIUJUKUNAULUM $.
+      ( vw vz weq wn wex wal wel wa el ax-nul sp wi wtru nfvd ax7 con3d spimedv
+      mptru eximii exdistrv mpbir2an ax9 com12 2eximi equequ2 notbid syl6bi a1d
```

Comment:
based on the introduction of wtru nfvd and mptru, would this proof be shortened by bj-spimev --> spimev?

Reply:
For some reason the minimizer doesn't find it, but I was able to manually get this:
<pre>
Proof of "dtru":
---------Clip out the proof below this line to put it in the source file:
      ( vw vz weq wn wex wal wel wa el ax-nul sp eximii exdistrv mpbir2an ax7v1
      nfv con3d spimefv ax9v2 con3dimp 2eximi equequ2 notbid syl6bi a1d pm2.61i
      com12 wi exlimivv mp2b exnal mpbi ) ABEZFZAGZUOAHFACIZADIZFZJZDGCGZCDEZFZ
      DGCGUQVBURCGUTDGACKUTAHUTDDALUTAMNURUTCDOPVAVDCDURVCUSVCURUSCDAUAUIUBUCVD
      UQCDDBEZVDUQUJVEVDCBEZFZUQVEVCVFDBCUDUEVGUPACVGARACEUOVFACBQSTUFVEFZUQVDV
      HUPADVHARADEUOVEADBQSTUGUHUKULUOAUMUN $.
---------The proof of "dtru" (404 bytes) ends above this line.
</pre>

I arbitrarily renamed `bj-spimev` into `spimefv` because `spimev` already exists, how should I rename `bj-spimev`?

Reply:
That's very nice ! You may add a "revised by" tag if you want.
I didn't expect this would affect 65 theorems !

Reply:
Ok, but how do I rename `bj-spimev`

Reply:
your proposal spimefv looks good to me

Reply:
Should proof modification of former `bj-spimedv` and `bj-spimev` stay discouraged?

Reply:
Sorry I overlooked that. If they are in Main, now that automatic minimization cannot add axiom dependencies, you can remove the discouragement tags.